source/guide/index.md

```diff
@@ -6,99 +6,156 @@ order: 1.1
+
+Currently, the `weex-toolkit` only supports to create the Vue.js project. The `rax-cli` maybe helpful if you want to use Rax. Please visit [Rax's official website](https://alibaba.github.io/rax/) for more details.
 
-### Setup
+### Set up
 
 Make sure you have already installed the [Node.js](https://nodejs.org/), and then install the `weex-toolkit` globally.
 
 ```bash
 npm install weex-toolkit -g
 ```
 
-It will add a `weex` command to your global path, run `weex --help` can see the manual of all its sub commands. The most useful command is `weex create [project-name]`:
+It will add the `weex` command to your global path. Then you can use `weex create` to create a starter project:
 
 ```bash
 weex create awesome-app
 ```
 
-After doing that, you'll get a standard **Weex + Vue.js** project under the `awesome-app` folder.
-
-> Currently, the `weex-toolkit` only supports to create Vue.js project. If you want to use Rax, the `rax-cli` maybe helpful, please go to [Rax's official website](https://alibaba.github.io/rax/) for more details.
+After doing that, you'll get a standard **Weex + Vue.js** project under the `awesome-app` folder of current path.
 
 ### Develop
 
+The next step is get into the path, install dependencies, and start.
 
 ```bash
 cd awesome-app
 npm install
-npm run start
+npm start
 ```
 
-It will start a web server on the `8080` port. You can open `http://localhost:8080/index.html` to see the result rendered on the web. Besides, You can also open the `http://localhost:8080/preview.html` to preview the web render result in iframe and with a QR code generated on the right, which you can use the [Weex plaground app]() to scan it to see the render result on the mobile.
+It will start a web server on the `8081` port.
+
+You can open `http://localhost:8081` to see the render result on the web. Besides, You can also open `http://localhost:8081/preview.html` to preview the web render result in iframe and with a QR code generated on the right. Use the [Weex playground app](http://weex-project.io/playground.html) to scan it and you'll see the render result on the phone.
 
 ![Preview]()
 
+The source code is located in `src/` folder. You can develop it as normal Vue.js project.
+
 ### Build and Run
 
+By default, the `weex create` command doesn't create the iOS and Android project, but you can use `weex platform add` to add them.
+
 ```bash
 weex platform add ios
 weex platform add android
 ```
 
+Depends on your network environment, it may take a while, please be patient.
+
+In order to develop the app on your local machine, you need to set up mobile development environments. For iOS, you should install [Xcode](https://developer.apple.com/xcode/). And for Android, you should install [Android Studio](https://developer.android.com/studio/index.html). If the development environments are ready, run the commands below could launch your app on the simulator or real device.
```

Comment:
When the development environments are ready,
run the commands below to launch your app on the simulator or the device.